examples/module-lifecycle/module_lifecycle.md

```diff
@@ -1,9 +1,7 @@
 # Module lifecycle
 
-All modules are initialized at program startup. Module initialization is ordered so that if module A imports module B,
-then module A is initialized after module B. The initialization phase ends by calling the `main` function if there is one. <br/><br/>
-A module's listeners are registered during module initialization.
-If there are registered listeners, then the initialization phase is followed by the listening phase. <br/><br/>
+All modules are initialized at program startup. Module initialization is ordered so that if module A imports module B, then module A is initialized after module B. The initialization phase ends by calling the `main` function if there is one.
```

Comment:
```suggestion
All modules are initialized at the program startup. Module initialization is ordered so that if `module A` imports `module B`, then `module A` is initialized after `module B`. The initialization phase ends by calling the `main` function if there is one.
```